spring/boot2-webflux-autoconfigure/src/main/java/com/linecorp/armeria/spring/web/reactive/ArmeriaServerHttpRequest.java

```diff
@@ -67,6 +69,12 @@ final class ArmeriaServerHttpRequest extends AbstractServerHttpRequest {
                    .publishOn(Schedulers.fromExecutor(ctx.eventLoop()));
     }
 
+    private static HttpHeaders springHeaders(RequestHeaders headers) {
+        final HttpHeaders springHeaders = new HttpHeaders();
+        toHttp1Headers(headers, (key, value) -> springHeaders.add(key.toString(), value));
```

Comment:
How about adding a new functional interface that takes three arguments and injecting the `springHeaders` into the method so that the lambda does not capture the object outside of the local scope.
```suggestion
        toHttp1Headers(headers, springHeaders, (output, key, value) -> output.add(key.toString(), value));
```

Reply:
That's brilliant. 👍